src/Assetic/Filter/CoffeeScriptFilter.php

```diff
@@ -64,10 +59,21 @@ public function filterLoad(AssetInterface $asset)
             $pb->add('--no-header');
         }
 
-        $pb->add($input);
+        // compile to JavaScript, otherwise coffee will run the script
+        $pb->add('--compile');
+
+        // we want it to print the output rather than run it
+        $pb->add('--print');
+
+        // read content from standard input
+        $pb->add("--stdio");
```

Comment:
I believe this was due to an older version of `symfony/process`, which now uses `setInput` rather than the previous `setStdin`. I haven't worked on Windows in a long time though so may be wrong.